src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -168,23 +168,28 @@ At this point, you now have the Dockstore CLI set up for interacting with the Do
 
 #### Part 6 - Install cwltool (Optional)
 Dockstore relies on [cwltool](https://github.com/common-workflow-language/cwltool) - a reference implementation of CWL - for local execution of tools and workflows described with CWL.
-You'll need to have Python 3 and [pip3](https://pip.pypa.io/en/latest/installing/) to be installed on your machine.
+You'll need to have Python 3 and [pipx](https://pipx.pypa.io/latest/installation/) to be installed on your machine.
 
 **Note:** cwltool must be available on your PATH for the Dockstore CLI to find it.
 
 You can install the version of cwltool that we've tested for use with Dockstore using the following commands:
 1. Install cwltool
 \`\`\`
 curl -o requirements.txt "${this.dsServerURI}/metadata/runner_dependencies?client_version=${this.dockstoreVersion}&python_version=3"
-pip3 install -r requirements.txt
+pipx install cwltool==${this.cwltoolVersion}
```

Comment:
Oddly enough, I tried this `pipx install` command,  but messed up the  following `runpip` command. 
Curious,  after putting things on the path with that `source` command, the commands in section 2 worked. 
i.e. the following all seemed successful. 
What's the deal with the `pipx runpip cwltool install -r requirements.txt` bit?

In other words, the following seems to result in a working cwltool. 
What does `pipx runpip cwltool install -r requirements.txt do?

(could be code comment or tutorial comment)

```
docker run -ti ubuntu:24.04
root@f20920837ae3:/# history
    1  apt update
    2  apt install curl wget pipx
    3  pipx install cwltool==3.1.20240708091337
    4  pipx ensurepath
    5  source ~/.bashrc
    6  pipx runpip cwltool list
    7  history
```

Reply:
That part forces installing the requirements we provide in requirements.txt. Otherwise it picks whatever is pinned via the cwltool version that's installed. So our attempt to make a more reproducible environment is sometimes error prone I would say.

Reply:
Ah ok, so specifying the cwltool version is not enough on it's own.

 Could be code comment, like
`pipx runpip cwltool install -r requirements.txt # this ensures that your version of cwltool and it's dependencies matches what we test with` or similar.  
Don't feel strongly

Reply:
I was about to ask the same questions as a Python newbie, then saw this thread... I second the suggestion of a comment explaining why we're installing additional dependencies after running `pipx install cwltool`; it seemed wrong to me at first.